src/gui/TimeLineWidget.cpp

```diff
@@ -140,9 +140,9 @@ void TimeLineWidget::addToolButtons( QToolBar * _tool_bar )
 					tr( "After stopping keep position" ) );
 	connect( behaviourAtStop, SIGNAL( changedState( int ) ), this,
 					SLOT( toggleBehaviourAtStop( int ) ) );
+	connect( this, SIGNAL( loadBehaviourAtStop( int ) ), behaviourAtStop,
+					SLOT( changeState( int ) ) );
 	behaviourAtStop->changeState( 1 );
```

Comment:
Please change this to use the the enum name (BackToZero, BackToStart, or KeepStopPosition) rather than just an int.

Reply:
I assume you meant `behaviourAtStop->changeState( 1 );`? I've changed that now.

Passing an int in those functions is the same as the line above, which wasn't written by me, and I left it as int for the sake of consistency.

Reply:
Ah yeah, clicked the wrong line haha. Thanks for fixing it!